apcd-cms/src/apps/submitter_renewals_listing/templates/view_submitter_registration_modal.html

```diff
@@ -0,0 +1,115 @@
+<div id="viewRegistrationModal_{{r.reg_id}}" class="modal fade" role="dialog">
```

Comment:
This is a copy of `view-registration-modal.html`. I think we can instead use that template and load it here and for the admin list registrations table, for maintainability.